graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/exceptions/DefaultDataFetcherExceptionHandler.kt

```diff
@@ -62,6 +56,21 @@ class DefaultDataFetcherExceptionHandler : DataFetcherExceptionHandler {
     }
 
     companion object {
+
         private val logger: Logger = LoggerFactory.getLogger(DefaultDataFetcherExceptionHandler::class.java)
+
+        private val springSecurityAvailable: Boolean by lazy {
```

Comment:
@paulbakker @srinivasankavitha I'm updating how we calculate this as well such that the assertion that if SpringSecurity is available only happens ounce.

Reply:
yeah that's a good improvement